docs/integration-test-helpers/masonry/MasonryContainer.js

```diff
@@ -125,7 +125,7 @@ export default class MasonryContainer extends Component<Props, State> {
 
   handleAddItems: () => void = () => {
     const { items } = this.state;
-    this.loadItems({ name: 'Manual Fetch Pin', from: items.length, force: true });
+    this.customLoadItems?.({ name: 'Manual Fetch Pin', from: items.length, force: true });
```

Comment:
This is a method defined on the class, it will always be defined — no need for optional chaining